public/static/js/projects.js

```diff
@@ -563,17 +568,29 @@ mciModule.controller('ProjectCtrl', function ($scope, $window, $http, $location,
     $scope.settingsFormData.commit_queue_aliases = $scope.settingsFormData.commit_queue_aliases.concat([item]);
     delete $scope.commit_queue_alias;
     $scope.invalidCommitQueuePatchDefinitionMessage = "";
-
   };
 
+  $scope.addGitTagAlias = function() {
+    if (!$scope.validGitTagVersionDefinition($scope.git_tag_alias)) {
+      $scope.invalidGitTagAliasMessage = "An alias must have a git tag alias, variant regex, and exactly one of task regex or tag";
+      return;
+    }
+    item = Object.assign({}, $scope.git_tag_alias);
+    item["alias"] = "__git_tag";
```

Comment:
I would only access properties like this if it's a variable. item.alias is easier to read